iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/iothubservices/FileUploadAndroidRunner.java

```diff
@@ -17,10 +17,10 @@
 import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.After;
 
 import java.io.IOException;
 
-@TestGroupB
```

Comment:
why remove?  rebalancing on the time execution between Android devices??

Reply:
Yes. With my other changes, the B group takes the longest, whereas the other group (not a, b, or c) finishes very quickly. This will help make a better balanced load on our 4 build devices